### ISSUE TITLE
failed Github authentication does not indicate an error

### ISSUE THREAD

**Issue by jack-kerouac** [dolbyzerr/grunt-github-releaser#3]
when authentication at Github fails (or any potential other error when GETting releases), there is no error indicated.

**Comment by brianlucas** [comment]
Agreed, came in to see if there were open issues and found this.  +1 to know if github credentials are required, or (alternatively) if you can avoid using github credentials when you have you ssh key loaded properly in your environment for git auth.
